### DISCUSSION TITLE
What is your best performance using this layout? (cpm)

### DISCUSSION THREAD

**Discussion by asim215** [binarybottle/engram-layout#80] [Q&A]
I use monkey type to train myself to use this beautiful layout. I am curious which your best performance there?
I use 5 min settings in about year or more, so I get 306 cpm (characters per minute). When you look in the charts top people get 1000-1300 cpm with qwerty, but they probably using it from the beginning. I don't think such high speed even needed, it only matters of your thought.

**Comment by binarybottle** [comment]
Hello, @asim215 -- thank you for your interest in the Engram layout!
I developed this optimized layout for comfort, but it also happens to make it easy to execute fast keystroke transitions. I type deliberately in a relaxed manner to prevent repetitive strain injuries, so I wouldn't be the person to ask about personal speed typing records.

**Comment by binarybottle** [comment]
See https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo!
